### PR TITLE
Centos8

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
There is currently no ferm package in epel for CentOS 8 :cry:

File: Gemfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ def location_for(place, fake_version = nil)
 end
 
 group :test do
-  gem 'voxpupuli-test', '>= 1.0.0',  :require => false
```
Comment (Dan33l) [inline Gemfile:14]: Is is related to this change ?

Comment (bastelfreak) [reply Gemfile:14]: this is cherry-picked from modulesync and required to make rspec-puppet-facts work

**Comment by ghoneycutt** [comment]
@bastelfreak if this would work if a package is present, suggest just commenting the acceptance tests out with a note that we're waiting on epel packages and a link where you could find them to see if they have present and then merging this.

**Comment by Dan33l** [comment]
i am not a fan to comment acceptance tests because we will forget to uncomment it for a while

**Pull request closed without merging**